content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -26,157 +23,289 @@ Get an overview of PodPresets at
+Here is the configuration file for the PodPreset:
 
 {{< codenew file="podpreset/preset.yaml" >}}
 
-Create the PodPreset:
+In the configuration file, you can see that the preset has an environment variable definition called `DB_PORT`
+and a volume mount definition called `cache-volume` which is mounted under `/cache`. The `selector` specifies that
+the preset will act upon any Pod that is labeled `role:frontend`.
+
+Create the preset:
 
 ```shell
 kubectl apply -f https://k8s.io/examples/podpreset/preset.yaml
 ```
 
-Examine the created PodPreset:
+Verify that PodPreset has been created:
 
 ```shell
 kubectl get podpreset
 ```
 ```
-NAME             AGE
-allow-database   1m
+NAME             CREATED AT
+allow-database   2020-01-24T08:54:29Z
 ```
 
-The new PodPreset will act upon any pod that has label `role: frontend`.
+Here is the configuration file containing the definition of an applicable Pod:
 
 {{< codenew file="podpreset/pod.yaml" >}}
 
-Create a pod:
+Create the Pod:
 
 ```shell
 kubectl create -f https://k8s.io/examples/podpreset/pod.yaml
 ```
 
-List the running Pods:
+Verify that the Pod is running:
 
 ```shell
 kubectl get pods
 ```
+
+The output shows that the Pod is running:
+
 ```
 NAME      READY     STATUS    RESTARTS   AGE
 website   1/1       Running   0          4m
 ```
 
-**Pod spec after admission controller:**
-
-{{< codenew file="podpreset/merged.yaml" >}}
-
-To see above output, run the following command:
+View the Pod spec altered by the admission controller in order to see the effects of the preset
+having been applied:
 
 ```shell
 kubectl get pod website -o yaml
 ```
 
-## Pod Spec with ConfigMap Example
+{{< codenew file="podpreset/merged.yaml" >}}
 
-This is an example to show how a Pod spec is modified by the Pod Preset
-that defines a `ConfigMap` for Environment Variables.
+The `DB_PORT` environment variable, the `volumeMount` and the `podpreset.admission.kubernetes.io` annotation
+of the Pod verify that the preset has been applied.
 
-**User submitted pod spec:**
+## Pod spec with ConfigMap example
 
-{{< codenew file="podpreset/pod.yaml" >}}
+This is an example to show how a Pod spec is modified by a Pod preset
+that references a ConfigMap containing environment variables.
 
-**User submitted `ConfigMap`:**
+Here is the configuration file containing the definition of the ConfigMap:
 
 {{< codenew file="podpreset/configmap.yaml" >}}
 
-**Example Pod Preset:**
+Create the ConfigMap:
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/configmap.yaml
+```
+
+Here is the PodPreset configuration file referencing the ConfigMap:
 
 {{< codenew file="podpreset/allow-db.yaml" >}}
 
-**Pod spec after admission controller:**
+Create the preset that references the ConfigMap:
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/allow-db.yaml
+```
+
+And the configuration file containing the definition of an applicable Pod looks like this:
```

Comment:
```suggestion
The following manifest defines a Pod matching the PodPreset for this example:
```
?

Reply:
Your suggestion actually is easier to read and understand.